### PR TITLE
Translate getting-started-guides/coreos/bare_metal_offline.md into Ch…

### PR DESCRIPTION
**Pull request by xingzhou**
…inese

Translate docs/getting-started-guides/coreos/bare_metal_offline.md into Chinese

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4359)

### CODE REVIEW

**Comment by xingzhou** [comment]
@markthink, please help to review the translation

**Comment by markthink** [comment]
@pwittrock  I am from caicloud.io, it looks good to me.

File: cn/docs/getting-started-guides/coreos/bare_metal_offline.md [hunk 1]
```diff
@@ -0,0 +1,708 @@
...
+display boot.msg
+
+MENU TITLE Main Menu
+
+LABEL local
+        MENU LABEL Boot local hard drive
+        LOCALBOOT 0
+```
+
+至此，您应当已经配置好一个可用的PXELINUX环境用来运行CoreOS节点了，您可以通过在本地运行VirtualBox或者物理裸机来验证这一点。
```
Comment (tengqm) [inline cn/docs/getting-started-guides/coreos/bare_metal_offline.md]: "您可以通过在本地运行VirtualBox或者物理裸机来验证这一点"

这句的原文理解错了。原文的意思是用户可以使用VirtualBox或者在物理裸机上对PXELINUX环境所提供的服务进行验证。

File: cn/docs/getting-started-guides/coreos/bare_metal_offline.md [hunk 2]
```diff
@@ -0,0 +1,708 @@
...
+
+## 添加CoreOS至PXE
+
+本节将介绍如何在一个现有的PXELINUX环境上设置CoreOS镜像。
```
Comment (tengqm) [inline cn/docs/getting-started-guides/coreos/bare_metal_offline.md]: 原文“This section describes how to setup the CoreOS images to live alongside a pre-existing PXELINUX environment.”
是否这样翻译更合适？

本节描述在已有PXELINUX环境的前提下，如何配置CoreOS镜像与之并存。

File: cn/docs/getting-started-guides/coreos/bare_metal_offline.md [hunk 3]
```diff
@@ -0,0 +1,708 @@
...
+
+稍后我们将介绍节点的配置。
+
+## Kubernetes
+
+部署以上配置前，需要首先创建一个`etcd`主节点(master)。为了做到这一点，我们需要使用一个特殊的cloud-config.yaml来pxe CoreOS，可以有两种方式：
+1. 第一种方式是将云配置文件模版化，然后通过编程的方式为不同的集群提供不同的配置。
+2. 第二种方式是运行一个服务发现协议从而可以在云环境中做服务的自动发现。
+
+在本demo中，我们仅简单地创建一个静态的单一`etcd`服务器来运行Kubernetes并提供`etcd`的主要服务。
```
Comment (tengqm) [inline cn/docs/getting-started-guides/coreos/bare_metal_offline.md]: “demo" 还是翻译成”示例“好些

Comment (tengqm) [inline cn/docs/getting-started-guides/coreos/bare_metal_offline.md]: ”a static single etcd server to host our Kubernetes and etcd master servers.“

通过静态方式创建一个etcd服务器，用于运行Kubernetes主控组件，并用作etcd主节点

File: cn/docs/getting-started-guides/coreos/bare_metal_offline.md [hunk 4]
```diff
@@ -0,0 +1,708 @@
...
+
+由于我们的集群处于一个离线的环境中，所以大部分的CoreOS和Kubernetes帮助进程是受限的。为了完成部署，我们需要下载Kubernetes的各种可执行文件到本地然后再启动运行。
+
+一种简单的方案是在DHCP/TFTP主机上搭建一个简易的web服务器，从而环境中的CoreOS PXE机器可以从其上下载各种可执行文件。
+
+为了达到这一目标，我们将启动一个`apache`服务器并提供运行Kuberetes所需要的各种可执行文件。
+
```
Comment (tengqm) [inline cn/docs/getting-started-guides/coreos/bare_metal_offline.md]: 各种 -> 各个？

**Comment by xingzhou** [comment]
thanks for the review, as the patch is merged just now, I'll submit an update PR to update the doc.

**Pull request merged**